issue #51, add more VGS recordings, adjust recording names

cdmiller opened the pull request:
First draft of adding more VGS recordings to the VGS player for issue 51 (https://github.com/dhammaorg/dhara-website-issues/issues/51)

I did not test this code, since I don't (yet) have a way to run it locally.

@jasonhoward7 maybe you can give it try?

cdmiller committed:
issue #51, add more VGS recordings, adjust recording names

jjdunn commented:
@cdmiller I pulled this changeset into my local environment but the virtual-group-sitting page is not loading
 
I'm getting a browser console error:
```
Uncaught TypeError: can't access property "onchange", document.getElementById(...) is null
    onload https://dhara.local/os/virtual-group-sitting/:144
```

lines 143-153 of page-virtual-group-sitting.php:
```
            window.onload = function() {
                document.getElementById("session-chooser").onchange = function() {
                    changeAudioPlayerTo( this.selectedIndex );
                }
                document.getElementById("background-chooser").onchange = function() {
                    changeBackgroundTo( this.selectedIndex );
                }
                document.getElementById("text-chooser").onchange = function() {
                    changeTextTo( this.selectedIndex );
                }
            }
```

line 144 is `document.getElementById("session-chooser")...`

BTW the formatting of js content in this file is terrible.  if you make another commit please fix the indentation

jjdunn commented:
during the call we noticed Craig wrote PHP code in a javascript block.  he will fix this :-)  
and also fix the js formatting

cdmiller committed:
issue #51, fix javascript

jjdunn commented:
@cdmiller - new code looks fine, but I'm getting "blocked" for four of the files.  see attached screenshot.  I guess this is a permission issue in discourses.dhamma.org ?
<img width="1346" height="312" alt="dhara-vgs-51-blocked-resources" src="https://github.com/user-attachments/assets/b5d57d11-dc35-4429-81dd-8571f200a401" />

cdmiller committed:
issue #51, bug fix to remove extranious '<--' from old comments

cdmiller committed:
issue #51, bug fix to avoid null reference on load, if not yet logged in as dhammaworker

cdmiller commented:
> @cdmiller - new code looks fine, but I'm getting "blocked" for four of the files. see attached screenshot. I guess this is a permission issue in discourses.dhamma.org ? 

@jjdunn Please try again, I found and removed some syntax errors, and fixed login issue where there was a null reference until you logged in correctly. Seems to work for me without errors when I uploaded to [dev](https://dhara.dev.webhost2.dhamma.org/) (now removed).

jjdunn commented:
seems to be working ok now.  all the tracks are playing; no more js errors in console.

cdmiller committed:
issue #51, remove Kushinagar Short recording, Metta section is too long
